eosclubhouse/libquest.py

```diff
@@ -1653,16 +1658,29 @@ def step_app_not_installed(self):
         '''Step method that is executed to check if the corresponding app is installed.
 
         This method is launched before the actual quest is run. It will
-        guide the user through the installtion of the app through the
+        guide the user through the installation of the app through the
         software backend.
 
-        By default shows a message and then abort.
+        If __app_common_install_name__ is defined, it will be used to construct the install
+        question's string.
+        Failing that, if there is a quest-specific _NOTINSTALLED string available, it is used.
+        Finally, it falls back on the generic NOTINSTALLED string.
         '''
-        action = self.show_choices_message('NOQUEST_NOTINSTALLED',
+
+        if self.__app_common_install_name__ is not None:
+            install_msg = 'NOQUEST_' + self.__app_common_install_name__ + '_NOTINSTALLED'
+        else:
+            has_install_msg = self._get_message_info('{}_NOTINSTALLED'.format(self._qs_base_id))
+            if has_install_msg:
```

Comment:
Could be reduced to a one liner.  
`install_msg = 'NOTINSTALLED' if has_install_msg else 'NOQUEST_NOTINSTALLED_GENERIC'`